packages/db/src/db.ts

```diff
@@ -11,18 +12,6 @@ import {
 } from "@truffle/db/loaders/commands";
 import { toIdObject, NamedResource } from "@truffle/db/meta";
 
-interface IConfig {
-  contracts_build_directory: string;
-  contracts_directory: string;
-  working_directory?: string;
-  db?: {
```

Comment:
What happens to this db field though? I don't think the `@truffle/config` has this field, are we adding it there? Well, it has `db: { enabled: bool }` so I guess adapter could be added there if needed?

Reply:
Hm, what do you think? Worth making this change? I kind of figured we could let it slide for now.

Reply:
I like not having our own custom config and this is the only thing that stands out as different. I think it would be nice for the adapter field to be set in the user config if they don't want to use the default. So we can leave this out until we want to add that functionality?

Reply:
Maybe I'll call this a later task :)